<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenConfig</class>
 <widget class="QWidget" name="fenConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Préfèrences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons_kde.qrc">
    <normaloff>:/IconsKDE/32x32/categories/preferences-system.png</normaloff>:/IconsKDE/32x32/categories/preferences-system.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="2" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Archivage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="2">
       <widget class="QToolButton" name="btn_Parcourir">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboRatio">
        <item>
         <property name="text">
          <string>Toutes</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 sur 500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aucune</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Répertoire d'archivage:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="le_RepArchi"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Trames Archivées :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Nombre de jours à recharger : </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sp_NBJours">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnOk">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btn_Cancel">
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>229</height>
      </size>
     </property>
     <property name="title">
      <string>Connexion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rad_Serie">
           <property name="text">
            <string>Série</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rad_Udp">
           <property name="text">
            <string>Udp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_Serie">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Port COM</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Baudrate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="cb_Serial">
           <property name="minimumSize">
            <size>
             <width>130</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cb_Baudrate">
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_Udp">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Ip Capteur</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Port Udp</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="le_IpCapteur">
           <property name="inputMask">
            <string>999.999.999.999</string>
           </property>
           <property name="text">
            <string>192.168.0.28</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="sp_UDPPort">
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>50000</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>81</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons_kde.qrc"/>
 </resources>
 <connections/>
</ui>
